app/src/main/java/com/example/keybindhelper/dto/Group.java

```diff
@@ -148,6 +145,7 @@ public Group Clone() {
         return ret;
     }
 
+    @NonNull
```

Comment:
Added @NonNull since this was overriding another null variable
source: https://kotlinlang.org/docs/null-safety.html#collections-of-a-nullable-type